src/_content/posts/guide-to-america.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "A First-Timer's Guide to Navigating America"
+layout: post
+
+author: Drew Winstel
+category: General
+published_datetime: 2024-08-16 06:00:00-04:00
+cover:
+    url: /assets/img/blog/registration-desk.jpg
+    alt: "Volunteers and attendees at registration desk"
+
+---
+
+For many first-time attendees from abroad, DjangoCon US represents the first time for people entering the United States.
+
+Here's a quick guide to navigating life in the United States as a newcomer:
+
+## Tipping
+
+Tipping (gratuities) in the United States is rampant. Here's a rough idea of how much you'll be expected to add on top of your bill:
+
+- Restaurants: 20% for everything except for the big-name fast food restaurants (e.g. McDonald's, Wendy's, Burger King)
+   - Even some self-serve kiosk ordering places will try to ask for a tip! This money will theoretically go to the staff preparing and/or delivering your food, but it definitely feels odd.
+- Coffee shops: $1-2 per cup of coffee
+- Bars: $1-2 per beer/glass of wine, $2-4 per cocktail, 20% on bottles of wine
+- Rideshare and taxis: 20%
+- Hotel room cleaning: ~$5/day (leave it on your pillow, weirdly)
+- Hotel valet parking: ~$3 per time they retrieve your car
+
+## Identification/Purchasing Alcohol
+
+To purchase alcohol or enter places that predominantly serve alcohol, there's a very good chance you'll be required to present identification. You _may_ be able to get away with a national ID card, but your safest option is your passport. It needs to be the original; photocopies are not accepted.
+
+## Sales Tax
+
+Many countries have some form of value added tax or goods and services tax rolled into the retail price that you pay, so a bottle of water that's listed as costing $2 is exactly what you  pay. In the US, this is very much _not_ the case. Instead, each state charges its own sales tax, and in many states, counties and municipalities add their own tax on top of that. Some places charge different tax rates for different classes of goods, such as a discount for food.
+
+In Durham County, NC, the sales tax is a flat 7.5%. The city of Durham does not add its own sales tax. For example, you will pay $2.15 for the previous hypothetical $2 bottle of water.
+
+## Payments
+
+Almost every venue nowadays takes Visa, MasterCard, American Express, and usually Discover. Make sure you notify your bank before you are traveling that you are going to the United States so you don't set off any fraud alerts. In the US, this requires calling the customer service number printed on the back of the card, but overseas banks may differ.
+
```

Comment:
Always have some cash just in case of issues with card or other means of payment you have. You can come in with cash or you can choose to make a withdrawal at the airport ATM's.

Reply:
When making payments, it's often more cost-effective to use the local currency rather than converting through your card. Your bank typically offers better exchange rates than those provided by merchants. By choosing to pay in the local currency, you can potentially save money on transaction fees and benefit from more favorable exchange rates..

Reply:
So on this point, it happens when you are paying for something. Like say food, it will ask you to select the currency to use, that is where this comes in.

> When making payments, it's often more cost-effective to use the local currency rather than converting through your card. Your bank typically offers better exchange rates than those provided by merchants. By choosing to pay in the local currency, you can potentially save money on transaction fees and benefit from more favorable exchange rates..

Reply:
Good call.